packages/manager/modules/storage/src/netapp/netapp.template.html

```diff
@@ -61,9 +61,9 @@
                         <span
                             class="oui-badge"
                             data-ng-class="{
-                                'oui-badge_success': (['running'].includes($ctrl.service.status)),
-                                'oui-badge_warning': (['creating', 'reopenning', 'suspending', 'deleting'].includes($ctrl.service.status)),
-                                'oui-badge_error': (['suspended', 'deleted'].includes($ctrl.service.status)),
+                                'oui-badge_success': ($ctrl.isStatusIn(['running'])),
+                                'oui-badge_warning': ($ctrl.isStatusIn(['creating', 'reopenning', 'suspending', 'deleting'])),
```

Comment:
More better way is to create a class and have methods like isRunning(), isSuccess(), isError etc